docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -6,21 +6,22 @@ Host isolation allows you to isolate hosts from your network, blocking communica
 
 Isolated hosts, however, can still send data to {es} and {kib}. You can also create <<host-isolation-exceptions, host isolation exceptions>> for specific IP addresses that isolated hosts are still allowed to communicate with, even when blocked from the rest of your network.
 
-Host isolation is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. 
-
-[NOTE] 
-=========================
-For {stack} version >= 7.15.0, host isolation is supported for endpoints running Windows, macOS, and these Linux distributions:
-
-* CentOS/RHEL 8
-* Debian 11
-* Ubuntu 18.04
-* Ubuntu 20.04
-* Ubuntu 22.04
-* AWS Linux 2
-
-To isolate and release hosts in any operating system, you must have the built-in `superuser` role. For more information, refer to {ref}/built-in-users.html[Built-in users].
-=========================
+.Requirements
+[sidebar]
+--
+* Host isolation is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. 
+
+* For {stack} version >= 7.15.0, host isolation is supported for endpoints running Windows, macOS, and these Linux distributions:
+
+** CentOS/RHEL 8
+** Debian 11
+** Ubuntu 18.04
```

Comment:
Could probably combine the three Ubuntu lines into one, e.g. "Ubuntu 18.04, 20.04, and 22.04"